config/desktop-development.json

```diff
@@ -135,15 +134,12 @@
 		"upgrades/checkout": true,
 		"upgrades/credit-cards": true,
 		"upgrades/domain-search": true,
-		"upgrades/in-app-purchase": false,
 		"upgrades/redirect-payments": true,
 		"upgrades/premium-themes": true,
 		"upgrades/precancellation-chat": true,
 		"upgrades/presale-chat": true,
 		"upgrades/upcoming-renewals-notices": true,
 		"upgrades/wpcom-monthly-plans": true,
-		"webpack/hot-loader": false,
-		"webpack/persistent-caching": false,
```

Comment:
Used for the last time in https://github.com/Automattic/wp-calypso/commit/d3c950e202#diff-11e9f7f953edc64ba14b0cc350ae7b9dL257